pkg/kn/commands/revision/delete.go

```diff
@@ -21,10 +21,14 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
-
 	"knative.dev/client/pkg/kn/commands"
+	v1 "knative.dev/client/pkg/serving/v1"
+	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 )
 
+// prune filter, used with "-p"
+var pruneFilter string
```

Comment:
Move into `NewRevisionDeleteCommand` scope? And per suggestion below as `pruneAll` as well.